etcdserver/api/v3compactor/periodic.go

```diff
@@ -136,13 +136,14 @@ func (pc *Periodic) Run() {
 				zap.Int64("revision", rev),
 				zap.Duration("compact-period", pc.period),
 			)
+			startTime := pc.clock.Now()
```

Comment:
Could we fix this in a separate PR?

Reply:
Done. 

https://github.com/etcd-io/etcd/pull/12303